rules/rom_e2e.bzl

```diff
@@ -0,0 +1,19 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:const.bzl", "CONST")
+load("@bazel_skylib//lib:structs.bzl", "structs")
+
+_ALL_LC_STATE_VALS = [lc_state_val for lc_state_val in structs.to_dict(CONST.LCV).values()]
+_SKIP_LC_STATE_VALS = [CONST.LCV.TEST_UNLOCKED0, CONST.LCV.PROD, CONST.LCV.PROD_END, CONST.LCV.DEV]
+_REM_LC_STATE_VALS = [lc_state_val for lc_state_val in _ALL_LC_STATE_VALS if lc_state_val not in _SKIP_LC_STATE_VALS]
+
+def maybe_skip_in_ci(tags, lc_state_val):
```

Comment:
Does this need tags as an input, or could it just take `lc_state_val` and return either `[]` or `["skip_in_ci"]`? The caller could just define their tags as `["tag1", ...] + maybe_skip_in_ci(lc_state_val)`.

Reply:
`+` doesn't work IIRC. I'll check again.

Reply:
Ah, this is actually what I was aiming for but I was either dreaming or accidentally wrote `["a"] + "b"` and thought that this operator was not supported. Thanks for making me try again!